GeneratorInterface/GenFilters/interface/MCParticlePairFilter.h

```diff
@@ -38,6 +38,9 @@
 namespace edm {
   class HepMCProduct;
 }
+namespace HepMC {
```

Comment:
Hi-
class FourVector; 

lets you reference pointers to your class. To return a full FourVector you need an include "FourVector.h" (or where-ever the class is defined). So, this header file should work the same without the class FourVector portion.